### ISSUE TITLE
Yet another hook request (and question about ssh)

### ISSUE THREAD

**Issue by ewancoder** [themoken/canto-next#13]
It would be great to have a hook which will work upon pressing 'g' (upon opening an article in a web-browser). Then one can form his own RSS feed based upon the items he reads (I am basically posting cool articles to my friends and it takes much time... it would be quite an optimisation to post them just an rss :D)

And I have question about ssh: it takes too long to get contents and everything lagging (over LAN). Should it be like this?
I did on server:
canto-daemon -p 31000 
I did on client:
canto-curses -a 192.168.100.33 -p 31000

(I am connecting just via LAN, not via ssh. The question is how to do it via ssh and will it be faster than lan and etc.)

UPD: And canto-curses sometimes outputs errno 111 - connection refused

**Comment by Narrat** [comment]
Could the info on the remote networking made be available again?
It got removed some time ago and I'm currently have problems with this.
Wanted to check, if I did something wrong with the configuration.

The problem I notice is, the daemon with -a "" and -p 11000, the daemon doesn't seem to fetch anything, unless a canto-curses session is running.

**Comment by ewancoder** [comment]
Sorry, I moved to inoreader and reading news on the go from my mobile phone now.

**Comment by Narrat** [comment]
No worries. I think I found the cause of my problem.

All is working well, as long an user who owns the canto user units is active on the pi (ssh or whatever)
And after reading those bits a few minutes ago:

```sh
Aug 25 01:59:25 Clio systemd[778]: Stopped Timer for forced updates of custom canto-daemon service - 1Hour
```
(A timer + service file which does every hour an forced update, as it seemed to help in the past, when the LED on the pi was having fun, and ssh into the pi reveaveled via htop, that the canto-daemon hogged quite some resources and tried to do something)

And the actual service for the daemon:
``` sh
 $ usystemctl status cantod             
● cantod.service - Canto RSS Daemon
   Loaded: loaded (/home/omnius/.config/systemd/user/cantod.service; enabled; vendor preset: enabled)
   Active: active (running) since Do 2016-08-25 12:24:32 CEST; 3min 32s ago
 Main PID: 1118 (canto-daemon)
   CGroup: /user.slice/user-1001.slice/user@1001.service/cantod.service
           └─1118 /usr/bin/python /usr/bin/canto-daemon -a -p 11000

Aug 25 12:24:32 Clio systemd[1108]: Started Canto RSS Daemon.
```

I closed the last ssh session at 1:59 and opened the new at 12:24.  
So this whole thing is handled by systemd the fact (which I kinda forgot) that the systemd --user instance is a per-user process, and not per-session.
Additionally as a feature which came with 230 
( https://github.com/systemd/systemd/blob/master/NEWS#L274 ), which would also be a candidate but as far I can tell is disabled on Arch and ALARM.
So I need to read about how to keep that stuff alive.
The nohup equivalent
